EDU-2517-become-isv (#386)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/marketplace/isv-signup.mdx

---
title: How to become an Independent Software Vendor on Azion Marketplace
description: Azion Marketplace is a curated digital catalog that allows companies to easily distribute their integrations to a wider audience.
meta_tags: vendor, edge computing, isv, marketplace
namespace: docs_marketplace_isv_signup
permalink: /documentation/products/marketplace/isv-signup/
---
import Button from '~/components/Button.astro';

> **Azion Marketplace** is a curated digital catalog that allows companies to easily distribute their integrations to a wider audience. This way, you can bring your product with a few clicks and accelerate the go-to-market process.

Reviewer: ```suggestion
**Azion Marketplace** is a curated digital catalog that allows companies to easily distribute their integrations to a wider audience. This way, you can bring your product to Azion with a few clicks and accelerate the go-to-market process.
```

<Button href="/en/documentation/products/marketplace/marketplace-seller-guide/" text="Marketplace Seller Guide" variant="secondary"> 

</Button>

---

## Requirements

- Have an [Azion account](/en/documentation/products/accounts/creating-account/).
- Complete the [sign-up and approval process](#signing-up-as-an-isv-on-azion-marketplace) to become a vendor.
- To be eligible to list your product with an associated charge, you must be able to receive payments from Azion and meet the following requirements:
  - Your organization needs to have *legal status in the United States*.
  - Your organization must have a *United States-based bank account*.

:::caution[Warning]
Azion Marketplace doesn't accept third-party companies reselling products.
:::
---

## Signing up as an ISV

To start your registration, you must fill in the [ISV application form](https://docs.google.com/forms/d/e/1FAIpQLScrOc_FyjVC11VFbVgDjuh6THh2jJs0-I8vduW0nZSCDGEPIg/viewform). This form requires you to provide key information about your organization and agree with the [Marketplace Agreement](/en/documentation/agreements/marketplace/marketplace-agreement/).

Fields marked with an asterisk are mandatory.

Where:

> | **Field** | **Description** |

Reviewer 2: precisa do negrito? acho que nunca padronizamos, né?

Reviewer: Acho que não, se não me engano o Astro já formata table heads pra negrito

> | **Field** | **Description** |

Author: ```suggestion
| Field | Description |
```

| --- | --- |

> | Email* | An email to contact you. It won't be visible to customers when your product is published in the Marketplace |

Reviewer 2: acho que todos deveria ser "on Marketplace".. pq é O marketplace da azion

> | Email* | An email to contact you. It won't be visible to customers when your product is published in the Marketplace |

Author: ```suggestion
| Email* | An email to contact you. It won't be visible to customers when your product is published on the Marketplace |
```

> | Contact Name | Name of the person responsible for the contract with Azion. It won't be visible to customers when your product is published in the Marketplace |

Author: ```suggestion
| Contact Name | Name of the person responsible for the contract with Azion. It won't be visible to customers when your product is published on Marketplace |
```

> | Company Name or Developer Name* | Name of the creator of your product. It'll be displayed when your product is published in the Marketplace |
> | Website | Your website. It'll be displayed when your product is published in the Marketplace |

Author: ```suggestion
| Company Name or Developer Name* | Name of the creator of your product. It'll be displayed when your product is published on Marketplace |
| Website | Your website. It'll be displayed when your product is published on Marketplace |
```

> | Company Logo | An URL containing the file of your logo that will be displayed when your product is published in the marketplace. It must be a `JPG` or `PNG` file with a maximum width of `120px` |

Reviewer 2: ```suggestion
| Company Logo | An URL containing the file of your logo that'll be displayed when your product is published in the Marketplace. It must be a `JPG` or `PNG` file with a maximum width of `120px` |
```

| Phone | A phone number to contact you. It won't be visible to customers |

> | Azion Marketplace Agreement* | The [Marketplace Agreement](/en/documentation/agreements/marketplace/marketplace-agreement/). You must read the document and mark the box to agree with the terms |

Reviewer 2: ```suggestion
| Azion Marketplace Agreement* | The [Marketplace Agreement](/en/documentation/agreements/marketplace/marketplace-agreement/). You must read the document and check the box to agree with the terms |
```

> When you've completed the form, click on the **Submit** button.

Reviewer 2: ```suggestion
Once you've completed the form, click on the **Submit** button.
```

The Azion team will receive your application form.

:::tip
Enable the **Send me a copy of my responses** switch to receive a copy of your application in the provided email.
:::

---

## Registering your product

> After finishing your signup, it's time to register your product. To do so, you must fill in the [product registration form](https://docs.google.com/forms/d/e/1FAIpQLSeoTbF2Qifr7qqHjoaxv9ujFB1Ak4xg6wqV0WfZfpBrpkQ12A/viewform). This form requires you to provide key details about your product and information that will be necessary to publish it on Azion Marketplace.

Reviewer 2: ```suggestion
After finishing your signup, it's time to register your product. To do so, you must fill in the [product registration form](https://docs.google.com/forms/d/e/1FAIpQLSeoTbF2Qifr7qqHjoaxv9ujFB1Ak4xg6wqV0WfZfpBrpkQ12A/viewform). This form requires you to provide key details about your product and information that'll be necessary to publish it on Azion Marketplace.
```

Fields marked with an asterisk are mandatory.

The form includes three sections:

### ISV's information

This section requires the ISV's name and a valid email to contact you as the responsible for the product.

Where:

> | **Field** | **Description** |

Author: ```suggestion
| Field | Description |
```

| --- | --- |
| Email* | An email to contact you as responsible for the product |

> | ISV Name* | Your name as the ISV and creator of the product. It's recommended to use the same in the **Company Name or Developer Name*** field in the [ISV application form](#signing-up-as-an-isv-on-azion-marketplace) |

Reviewer 2: ```suggestion
| ISV Name* | Your name as the ISV and creator of the product. It's recommended to use the same name as in the **Company Name or Developer Name*** field in the [ISV application form](#signing-up-as-an-isv-on-azion-marketplace) |
```

### Preparing your product

This section requires basic information to display on the page of your product in the Azion Marketplace.

Where:

| **Field** | **Description** |
| --- | --- |
| Product Name* | Name of your product that will be displayed on the card and on the product detail page in the Marketplace |
| Product Version* | The version of your product that will be available on the marketplace. Example `1.1.0` |
| Category* | The Azion Marketplace category that better fits your product. Select *at least one option* |

> | Pricing model* | The pricing model that better fits your product. You can choose between `Free` or `Bring your own license` models. More information on the [Marketplace Seller Guide](https://www.azion.com/en/documentation/products/marketplace/marketplace-seller-guide/) |

Reviewer 2: ```suggestion
| Pricing model* | The pricing model that better fits your product. You can choose between `Free` or `Bring your own license` models. Find more information on the [Marketplace Seller Guide](/en/documentation/products/marketplace/marketplace-seller-guide/) |
```

### Describing your product

This section requires additional information to be displayed on the page of your product in the Azion Marketplace, facilitating the understanding for customers.

Where:

> | **Field** | **Description** |

Author: ```suggestion
| Field | Description |
```

| --- | --- |
| Headline* | A summary of your product, up to `80 characters` |
| Overview* | A description of your product, up to `4,000 characters`. You can explain the purpose of your product and what use cases it aims to solve, as well as any link to additional content, such as documentation, videos, and websites, among others |

> | Usage information* | An explanation of how to use your product, including requirements and first steps. You can include additional content such as documentation, webinars, how-to, user agreements, and Terms of Service, among others |

Reviewer 2: ```suggestion
| Usage information* | An explanation of how to use your product, including requirements and first steps. You can include additional content such as documentation, webinars, how-to, user agreements, and terms of service, among others |
```

Reviewer 2: por ser geral

| Support* | A description of the support provided to this specific product, including your service channels |

> When you've completed the form, click on the **Submit** button.

Reviewer 2: ```suggestion
Once you've completed the form, click on the **Submit** button.
```

> When you've completed the form, click on the **Submit** button.

Reviewer: ```suggestion
When you've completed the form, click the **Submit** button.
```

The Azion team will receive the registration of your product.

---

## Receiving the approval

> The last stage is awaiting the approval. The Azion team will analyze your ISV application and your product registration to determine if everything is compliant and if your product is a good fit for Azion Marketplace.

Reviewer 2: ```suggestion
The last stage is awaiting the approval. The Azion team will analyze your ISV application and your product registration to determine if everything's compliant and if your product is a good fit for Azion Marketplace.
```

The team will contact you through the email you registered to confirm the approval and explain further steps to finish your product publication in Azion Marketplace.

:::tip

> For further information on requirements and conditions to be a Marketplace Seller, check the [Marketplace Seller Guide](https://www.azion.com/en/documentation/products/marketplace/marketplace-seller-guide/).

Reviewer 2: ```suggestion
For further information on requirements and conditions to be a Marketplace Seller, check the [Marketplace Seller Guide](/en/documentation/products/marketplace/marketplace-seller-guide/).
```

> For further information on requirements and conditions to be a Marketplace Seller, check the [Marketplace Seller Guide](https://www.azion.com/en/documentation/products/marketplace/marketplace-seller-guide/).

Reviewer: ```suggestion
For further information on requirements and conditions to be a Marketplace Seller, check the [Marketplace Seller Guide](/en/documentation/products/marketplace/marketplace-seller-guide/).
```
